packages/components-native/src/StatusIndicator/StatusIndicator.style.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from "react-native";
+import { tokens } from "../utils/design";
+
+const statusIndicatorDiameter = tokens["space-small"];
+
+export const styles = StyleSheet.create({
+  statusIndicator: {
+    borderRadius: tokens["radius-circle"],
+    backgroundColor: tokens["color-success"],
+    width: statusIndicatorDiameter,
+    height: statusIndicatorDiameter,
```

Comment:
Notice the margin is being removed, as we no longer required the very finicky 3px adjustment that was needed with the more square-ish indicator